docs/cloud/rahti/storage/expand-volumes.md

```diff
@@ -105,3 +95,16 @@ oc edit deploy/<name of deployment>
 * Finally scale up the deployment.
 
 In order to check the procedure worked, you may enter in a `Pod` that is mounting the volume and check the new size.
+
+## Dynamically
+
+If the volume was created with dynamic resize activated, it is as easy as editing the corresponding volume (`PVC` object) and change the size:
```

Comment:
Should we keep this section if dynamic resize is not enabled anyway?

Reply:
It is a very good question.